src/axolotl/loaders/patch_manager.py

```diff
@@ -234,7 +234,7 @@ def _apply_gemma3_conditional_generation_forward_patch(self):
 
     def _apply_sequence_parallel_patches(self):
         """Apply sequence parallelism patches."""
-        if self.cfg.sequence_parallel_degree > 1:
+        if self.cfg.sequence_parallel_degree and self.cfg.sequence_parallel_degree > 1:
```

Comment:
Technically we should fix our tests since this doesn't happen in practice :) but it's okay for now I suppose.